cfgov/v1/wagtail_hooks.py

```diff
@@ -67,15 +71,26 @@ def configure_page_revision(page, is_sharing, is_live):
 
 def flush_akamai(page, is_live):
     if is_live and settings.ENABLE_AKAMAI_CACHE_PURGE:
-        from publish_eccu.publish import publish as akamai_cache_reset
-
-        url_paths = [page.url_path.replace('cfgov/', '')]
-        if url_paths[0] == '/':
-            is_home_page = True
-        else:
-            is_home_page = False
-
-        akamai_cache_reset(url_paths, invalidate_root=is_home_page, user_email=page.owner.email)
+        akamai_user = os.environ.get('AKAMAI_USER')
+        akamai_pw = os.environ.get('AKAMAI_PASSWORD')
+        akamai_object_id = os.environ.get('AKAMAI_OBJECT_ID')
+        akamai_url = 'https://api.ccu.akamai.com/ccu/v2/queues/default'
+        headers = {'content-type': 'application/json'}
+        payload = {
+            'action': 'invalidate',
+            'type': 'cpcode',
+            'domain': 'production',
+            'objects': [akamai_object_id]
+        }
+        r = requests.post(
+            akamai_url,
+            headers=headers,
+            data=json.dumps(payload),
+            auth=(akamai_user, akamai_pw)
+        )
```

Comment:
Perhaps this is out of scope for this PR, but it might be nice to check `r.status_code` here to assert that the request was successful (from [the docs](https://api.ccu.akamai.com/ccu/v2/docs/#section_Basics)) it looks like it should return 201.